src/lucky_router/matcher.cr

```diff
@@ -16,32 +16,11 @@
 # router.match("get", "/users").payload # :index
 # ```
 class LuckyRouter::Matcher(T)
-  getter routes
   # aliases to help clarify what the @routes has is made of
   alias RoutePartsSize = Int32
   alias HttpMethod = String
-
-  # The matcher stores routes based on the HTTP method
-  #
-  # Each route key is a `Fragment(T)`. Where `T` is the type of the payload. See
-  # `Fragment` for details on how it works
-  #
-  # ## Example
-  #
-  # ```
-  # router = LuckyRouter::Matcher(Symbol).new
-  # router.add("get", "/users/:user_id", :index)
-  #
-  # # Will make @routes look like:
-
-  # {
-  #   "get" => Fragment(T) # The fragment for this route
-  # }
-  # ```
-  #
-  # So if trying to match a POST request it will not even try because the request
-  # method does not match any of the known routes.
-  @routes = Hash(HttpMethod, Fragment(T)).new
```

Comment:
If you could add a comment similar to this to help readers visualize how things are mapped to the router internals that'd be awesome. Could be done in a separate PR anytime. But this isn't a blocker so no worries if you don't have time. C